### DISCUSSION TITLE
Accuracy of the Hubble Constant Tested with High Resolution Observations of Cepheids by JWST

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#117] [Redshift Models]
"Crowded No More: The Accuracy of the Hubble Constant Tested with High Resolution Observations of Cepheids by JWST"
[https://arxiv.org/abs/2307.15806](https://arxiv.org/abs/2307.15806)

Cepheids observed with JWST provide further support to the claim "that systematic errors in HST Cepheid photometry do not play a significant role in the present Hubble Tension."

And it also supports the claim that "the CMB is not the heat leftover from the Big Bang".

**Comment by budrap00** [comment]
Louis,

Why do you think this finding, which to me seems to support the Hubble tension being real, also supports the claim that "the CMB is not the heat leftover from the Big Bang"?

**Reply by RedshiftDrift** [reply]
More measurements and calibrations of the Hubble constant by Riess *et al.* give $H_0(\mbox{SNe}) = 73.4$ km/s/Mpc.
Riess *et al.* still can't identify any systematic error, strenghtening the significance of $H_0(\mbox{SNe})$.

If the CMB was produced by heat leftover from the Big Bang, $H_0(\mbox{SNe})$ should be equal to $H_0(\mbox{CMB}) = 67.4$ km/s/Mpc derived from the CMB and the ΛCDM model.  The values disagree, hence the test by Riess *et al.* weakens the significance of $H_0(\mbox{CMB})$.

If the tension is real, it is more likely linked to an error on $H_0(\mbox{CMB})$.  One way to explain that error is that there was no Big Bang and no heat leftover to produce the CMB, and the value of $H_0(\mbox{CMB})$ is meaningless.

**Reply by budrap00** [reply]
Yeah, I see your point but if history holds there will be a serious reconsideration of one or more parameters or the addition of some new Dark Sauce. Everything will be made to come out right and 67.4 will be declared a successful prediction of the model. That's what always happens isn't it. Do you see this time being different for some reason?

**Reply by RedshiftDrift** [reply]
This time is not different, but Riess will measure $H_0 = 73.4$ m/s/kpc and maintain the tension for as long as he is an active astronomer.

For tired-light models the value $H_z = H_0/c = 0.245$/Gpc obtained by Riess is the correct one for our region of space, so any work measuring this number is useful.

**Reply by budrap00** [reply]
Hi Louis,

My mistake. I agree that Reiss' results look robust because he is measuring distance independently of redshift - his redshift-distance relationship is useful and can be separated from the velocity interpretation. I meant to say that the Planck result of  67.4 is a model dependent inference since neither redshift nor distance to the source are directly measured values. It is that value that will be either modified to fit Reiss, or a cause (Dark Sauce) for the discrepancy will be postulated and subsequently declared a feature of the Cosmos ala DM and DE. 

I don't think it is possible to be too cynical re the self-reinforcing, delusional nature of the standard model proponent's belief system. Until these jokers are defunded there will be no rational scientific cosmology on offer from the academic community.

**Comment by VitalyPetrikov** [comment]
Regarding arxiv.org/abs/2307.15806.
Louis, I don't trust the ability of NASA astronomers to correctly interpret JWST measurements. All NASA astronomers stick to BB or they won't get support. They do not give any specifics about the expansion of space (the universe). Only vague statements. Nothing prevents them from manipulating the expansion of the universe (space) for the sake of the constancy or accuracy of the Hubble parameter.

**Reply by RedshiftDrift** [reply]
I agree many interpretations are not reliable, but sharp features on the spectrum are difficult to interpret incorrectly, like a spectroscopic redshift of the sharp cutoff (Lyman break) or of spectroscopic lines.

**Comment by VitalyPetrikov** [comment]
Yes, it's all true. But the Lyman dips (and the like) do not violate the Hubble parameter variability principle.
In general, the question is posed as follows: either the expansion of space exists, in which case the Hubble parameter is a constant, or the Hubble parameter is variable, in which case the expansion of space is redundant. At the same time, the recession of cosmic galaxies with the expansion of the Universe (space) should not be combined. These are different categories.
In my opinion, on this border the division of supporters and opponents of the Big Bang occurs.this border.
